src/appleseed.studio/mainwindow/rendering/lightpathstab.cpp

```diff
@@ -175,11 +175,13 @@ void LightPathsTab::create_light_paths_widget()
 {
     // Create the OpenGL widget.
     const CanvasProperties& props = m_project.get_frame()->image().properties();
+    const unsigned int msaa_samples = 4; // TODO: Allow configuration?
```

Comment:
Use `size_t` by default for unsigned types, unless you need compactness in which case you should use the fixed size types in `foundation`. But here `size_t` seems like a good fit.

Spell "true" constants in PascalCase, capitalizing abbreviations: `MSAASamples`. Eventually those "true" constants will be turned into `constexpr` when that will be supported by all compilers we are required to use (VS 2012 being the last ancient one).

Finally, I think this constant should be defined in `LightPathsWidget`, it doesn't feel like it belongs in this file.

Reply:
Yeah, I think I'm going to leave most of the msaa logic out until I can get it actually working. The idea was that this should be configurable outside `LightPathsWidget` so you could set it in preferences or something, tough, eventually.